#! /usr/bin/env python

PACKAGE='mbzirc_task2_perception'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("section_size", double_t, 0, "extract section_size * sigma section", 2.0, 0.0, 5.0)
gen.add("screen_debug", bool_t, 0, "screen_debug", True)

exit(gen.generate(PACKAGE, "mbzirc_task2_perception", "InsigmaNormalizer"))
